### PR TITLE
Added procedure to equip spells equipping by rule

### PR DESCRIPTION
**Pull request by TheOnePharaoh**
Added procedure to equip spells equipping by rule 
Function By @MichaelLawrenceDee
this function is to be added to the equip cards instead of the standard effects 
**`Auxiliary.AddEquipProcedure(card c,int player p,function f,function eqlimit,fuction cost,function tg,function op)`**

Scripts Provided By @Shad3Light

km_

km_2

[http://www.db.yugioh-card.com/yugiohdb/faq_search.action?ope=5&fid=17489&keyword=&tag=-1&request_locale=ja](http://www.db.yugioh-card.com/yugiohdb/faq_search.action?ope=5&fid=17489&keyword=&tag=-1&request_locale=ja)

### CODE REVIEW

File: c11913700.lua [hunk 1]
```diff
@@ -68,11 +37,7 @@ function c11913700.spop(e,tp,eg,ep,ev,re,r,rp)
 	if Duel.GetLocationCount(tp,LOCATION_MZONE)<=0 then return end
 	Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_SPSUMMON)
 	local g=Duel.SelectMatchingCard(tp,c11913700.spfilter,tp,0x13,0,1,1,nil,e,tp)
-	if g:GetCount()>0 then
-		if g:GetFirst():IsHasEffect(EFFECT_NECRO_VALLEY) and Duel.IsChainDisablable(0) then
-			Duel.NegateEffect(0)
-			return
-		end
+	if g:GetCount()>0 and not g:GetFirst():IsHasEffect(EFFECT_NECRO_VALLEY) then
```
Comment (nekrozar) [inline c11913700.lua]: Why change you this?

Comment (Shad3Light) [inline c11913700.lua]: Hmm. I don't remember changing anything other than the equipping parts.
The original commit is against Percy's scripts, so maybe this one was outdated.

**Comment by TheOnePharaoh** [comment]
@nekrozar Fixed Outdated Scripts
Please merge before other updates cause more conflicts

**Comment by Hello23-Ygopro** [comment]
I think nekrozar doesnt have write access , only DailyShana and a few other people..
but i dont think this will ever get merged sorry

**Comment by TheOnePharaoh** [comment]
@DailyShana Please check this out

**Comment by TheOnePharaoh** [comment]
sorry I forgot to add the screens and the link before.

**Comment by mercury233** [comment]
Sorry, checking this may take much time, and I will spend my time on something more urgently.

**Comment by MichaelLawrenceDee** [comment]
Hmm.... Bashing Shield

**Pull request closed without merging**